core/src/main/java/org/apache/calcite/sql/type/SqlTypeTransforms.java

```diff
@@ -96,6 +96,21 @@ public abstract class SqlTypeTransforms {
         return typeToTransform;
       };
 
+  /**
+   * Parameter type-inference transform strategy where a derived type is
+   * transformed into the same type, but nullable if and only if the type
+   * of a call's operand #0 (0-based) is nullable.
+   */
+  public static final SqlTypeTransform ARG0_NULLABLE =
+      (opBinding, typeToTransform) -> {
+        RelDataType arg0 = opBinding.collectOperandTypes().get(0);
```

Comment:
similar comment to what i left on your other PR opBinding.getOperandType()

Reply:
fixed